src/components/ScholarshipForm.jsx

```diff
@@ -8,13 +8,43 @@ import {
   StepContent,
   Grid,
   Typography,
+  FormControlLabel,
+  Checkbox,
 } from '@material-ui/core';
 import { makeStyles } from '@material-ui/core/styles';
 import PropTypes from 'prop-types';
 import validationSchema from '../validation/ValidationSchema';
 import ScholarshipAmountField from './ScholarshipAmountField';
 import DatePicker from './DatePicker';
 import FormikTextField from './FormikTextField';
+import FormikMultiSelect from './FormikMultiSelect';
+import FormikAutocomplete from './FormikAutocomplete';
+import { SCHOOLS, STATES, MAJORS } from '../types/options';
+import GradeLevel from '../types/GradeLevel';
+import Ethnicity from '../types/Ethnicity';
+import experiments from '../lib/experiments';
+
+const gradeOptions = {
+  'Middle School': GradeLevel.MiddleSchool,
```

Comment:
(for a follow-up PR)

I recommend having these string representations be fetched from a util class method or a method in the same namespace:
https://stackoverflow.com/a/28151986

Then you could dynamically generate this list.

Reply:
interesting 🤔